resources/assets/js/components/ModelLoad.vue

```diff
@@ -1,19 +1,72 @@
 <template>
-  <div>
-    <input type="number" id="inputModelID" name="inputModelID" v-model="modelID" @keyup.enter="modelLoad">
-    <button type="button" class="btn btn-primary ml-2" @click="modelLoad">Load Model</button>
-  </div>
+
+    <div>
+
+        <!-- Trigger the modal with a button -->
+        <button type="button" class="btn btn-primary ml-2" data-toggle="modal" data-target="#myModal" @click="loadModelEvidencio">Search Evidencio Model</button>
+
+        <!-- Modal -->
+        <div id="myModal" class="modal fade" role="dialog">
+            <div class="modal-dialog modal-lg">
+
+                <!-- Modal content-->
+                <div class="modal-content">
+                    <div class="modal-header">
+                      <input type="text" name="inputModelSearch" v-model="modelSearch" class="form-control" style="width:100%; font-size:x-large; height:50px;" placeholder="Search for Model..." v-on:keyup.enter="loadModelEvidencio" autofocus>
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                    </div>
+
+                    <div class="modal-body">
+                        <div class="list-group" id="list-tab" role="tablist">
+                            <a class="list-group-item list-group-item-action"  v-for="(search, index) in searchs" :key="index" v-if="search.title" v-text="search.title" @click="modelLoad(search.id)"
+                                data-dismiss="modal"></a>
+                        </div>
+                    </div>
+                    <div class="modal-footer">
+                        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
+
 </template>
 
 <script>
 export default {
   data() {
     return {
-      modelID: 0
+      modelSearch: "",
+      modelID: 0,
+      searchs: {
+        type: Object,
+        default: () => {}
+      }
     };
   },
   methods: {
-    modelLoad() {
+    loadModelEvidencio() {
+      var self = this;
+
+      $.ajax({
```

Comment:
You might want to use axios for better ajax requests. You can also add the CSRF token by default, so that you do not have to add it manually all the time.

Reply:
`Axios` is nice indeed, works well with `Vue`. I am still starting with it though (yesterday, actually), so I still haven't quite figured out how it deals with multiple AJAX requests with one final callback.